README.md

```diff
@@ -102,9 +102,19 @@ Eg:
 -Dgo.plugin.build.status.gerrit.codeReviewLabel=Verified
 ```
 
+#### Gitlab
+**Setup:**
+- You need to provide `endpoint`, `oauth token` through system property `go.plugin.build.status.gitlab.endpoint`, `go.plugin.build.status.gitlab.oauth`.
+Eg:
+```
+-Dgo.plugin.build.status.gitlab.endpoint=http://gitlab.com
+-Dgo.plugin.build.status.gitlab.oauth=XXXX
+```
+
 ## FAQs
 
 [1]: images/list-plugin.png  "List Plugin"
 [2]: images/configure-plugin.png  "Configure Plugin"
-[3]: images/pipeline-schedule.png  "Pipeline Schedule"
-[4]: images/update-status.png  "On Successful Status Update"
+[3]: images/gitlab-plugin-settings.png  "Configure Gitlab Plugin"
+[4]: images/pipeline-schedule.png  "Pipeline Schedule"
```

Comment:
The images seem to be swapped out. See the image for point `When the stage status changes` and `The PR state is updated` [here](https://github.com/ckaushik/gocd-build-status-notifier/blob/master/README.md) and [here](https://github.com/gocd-contrib/gocd-build-status-notifier/blob/master/README.md)

Reply:
fixed this